WireUI/Sources/WireSidebar/Views/Preview/SidebarMenuItemPreview.swift

```diff
@@ -0,0 +1,82 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+
+struct SidebarMenuItemPreview: View {
+
+    @State private var topMenuIconSize: CGSize?
+    @State private var bottomMenuIconSize: CGSize?
+
+    var body: some View {
+        VStack {
+            // Display two separate menus here in order to verify, that the size
+            // (especially the width) of the icons is equal for all items within their menu only.
+            SidebarMenuItemContainer(iconSize: $topMenuIconSize) { iconSize in
+                SidebarMenuItemView(icon: "text.bubble", iconSize: iconSize, isHighlighted: false, title: { Text("Regular") }, action: { print("show all conversations") })
+                SidebarMenuItemView(icon: "gamecontroller", iconSize: iconSize, isHighlighted: true, title: { Text("Initially highlighted") }, action: { print("show all conversations") })
+                SidebarMenuItemView(icon: "person.3", iconSize: iconSize, isLink: true, title: { Text("Initially highlighted") }, action: { print("show all conversations") })
+            }
+            .overlay {
```

Comment:
nit-pick: especially in switchUI views, I prefer having each param returning to the next line. Just sharing a thought about formatting, it will be easier to fix when swift format is in place (aka don't fix all your views yet)

```suggestion
                SidebarMenuItemView(icon: "text.bubble", 
                                                       iconSize: iconSize,
                                                       isHighlighted: false,
                                                       title: { Text("Regular") }) { 
                                                          print("show all conversations") 
                                                       }
     ```

Reply:
Fine for me. I changed it for this file and will keep it in mind.

Reply:
Unfortunately I changed it on the wrong branch, now the changes are visible in this PR: https://github.com/wireapp/wire-ios/pull/1934/files